view/frontend/email/tesco_template.html

```diff
@@ -0,0 +1,17 @@
+<!--@subject {{trans "Tesco Lotus Payment"}}  @-->
+{{template config_path="design/email/header_template"}}
+<div>
+    <p>{{trans "%thankyou" thankyou="Thank you for your order."}}</p>
+    <p>{{trans "%message" message="This is your Tesco Lotus payment code. Please print it and bring it to any Tesco
+        Lotus store within the next 24 hours to finalise payment. After that time your order will be automatically
+        cancelled."}}</p>
+    <p>{{trans "%orderid" orderid="Order ID:"}} <strong>{{var data.orderId|raw}}</strong></p>
```

Comment:
@jacstn Just idea but I think this can be grouped in one `<p></p>` element (as result that there are big spaces between each line of this)
```html
<p>
    {{trans "%orderid" orderid="Order ID:"}} <strong>{{var data.orderId|raw}}</strong><br/>
    {{trans "%valid" valid="Valid until:"}} <strong>{{var data.valid|raw}}</strong><br/>
    {{trans "%amounttopay" amounttopay="Amount to pay:"}} <strong>{{var data.amount|raw}}</strong>
</p>
```
or `<ul><li></li></ul>` would be more appropriate?

Reply:
html emails have a lot of limitations in both html and CSS, that is why it is ok to leave `<p>`

Reply:
ok.. that is not email.. it is checkout template.. I will change it.. 👍

Reply:
ok.. for some _monday_ related reason I needed 3 times to read and understand what you mean...  I will change it.. 👍